packages/react/src/cunningham-tokens.css

```diff
@@ -112,6 +112,13 @@
 	--c--theme--breakpoints--lg: 992px;
 	--c--theme--breakpoints--xl: 1200px;
 	--c--theme--breakpoints--xxl: 1400px;
+	--c--components--toast--slide-in-duration: 1000ms;
+	--c--components--toast--slide-out-duration: 300ms;
+	--c--components--toast--background-color: var(--c--theme--colors--greyscale-000);
```

Comment:
On figma the greyscale-100 color is used. Is there any accessibility issue to use this color as background ?

Reply:
Oh you are right I missed it! I don't think there is any accessibility issue as the contrast remains relatively high, nope?